src/components/Kv/KvSocialShareButton.vue

```diff
@@ -42,17 +42,17 @@
 				<kv-button
 					variant="ghost"
 					class="social-button"
-					data-testid="share-twitter-button"
-					v-kv-track-event="[trackingCategory, 'share', 'twitter', utmCampaign, loanId]"
+					data-testid="share-bluesky-button"
+					v-kv-track-event="[trackingCategory, 'share', 'bluesky', utmCampaign, loanId]"
 					@click="showSharePopUp(
-						twitterShareUrl({utmCampaign, utmContent}),
-						'Thanks for tweeting!')"
+						blueskyShareUrl({utmCampaign, utmContent}),
+						'Thanks for sharing!')"
 				>
 					<kv-material-icon
-						class="social-button__icon social-button__icon--twitter"
-						:icon="mdiTwitter"
+						name="bluesky" title="Bluesky"
+						class="social-button__icon social-button__icon--bluesky"
```

Comment:
It doesn't look like this class currently exists, but maybe it will need to after using the `kv-icon` component here like in `SocialShareV2`.